bin/Powheg/run_pwg.py

```diff
@@ -1161,18 +1161,19 @@ def runhnnlo(folderName, njobs, QUEUE):
                 default_pdf = "320900"  # for 4 flavours
 
             for line in open(args.folderName+'/powheg.input') :
-                if 'lhans1' in line and len(line.split()) >= 2:
-                    test_pdf1 = line.split()[1].strip()
-                if 'lhans2' in line and len(line.split()) >= 2:
-                    test_pdf2 = line.split()[1].strip()
+                m_keyval = line.split()
```

Comment:
can you use something more intuitive than "m_keyval" ? Like "n_column" or similar